daprdocs/content/en/operations/support/support-preview-features.md

```diff
@@ -14,9 +14,31 @@ For CLI there is no explicit opt-in, just the version that this was first made a
 ## Current preview features
 
 | Feature | Description | Setting | Documentation | Version introduced |
-| ---------- |-------------|---------|---------------|-----------------|
+|---|---|---|---|---|
 | **`--image-registry`** flag in Dapr CLI| In self hosted mode you can set this flag to specify any private registry to pull the container images required to install Dapr| N/A | [CLI init command reference]({{<ref "dapr-init.md#self-hosted-environment" >}}) | v1.7 |
-| **Resiliency** | Allows configuring of fine-grained policies for retries, timeouts and circuitbreaking. | `Resiliency` | [Configure Resiliency Policies]({{<ref "resiliency-overview">}}) | v1.7|
 | **App Middleware** | Allow middleware components to be executed when making service-to-service calls | N/A | [App Middleware]({{<ref "middleware.md#app-middleware" >}}) | v1.9 |
+| **Streaming for HTTP service invocation** | Enables (partial) support for using streams in HTTP service invocation; see below for more details. | `ServiceInvocationStreaming` | &nbsp; | v1.10 |
 | **App health checks** | Allows configuring app health checks | `AppHealthCheck` | [App health checks]({{<ref "app-health.md" >}}) | v1.9 |
 | **Pluggable components** | Allows creating self-hosted gRPC-based components written in any language that supports gRPC. The following component APIs are supported: State stores, Pub/sub, Bindings | N/A | [Pluggable components concept]({{<ref "components-concept#pluggable-components" >}})| v1.9  |
+
+### Streaming for HTTP service invocation
```

Comment:
I've added this section to the same page, rather than creating a new page like the other preview features do. The reason is that the content in this section is specifically about the feature flag and how it influences Dapr's behavior, rather than the feature itself, which doesn't require any documentation on its own.

Because of that, it's meant to be temporary until the feature flag is removed too, which makes me feel a bit uncomfortable about creating a TOC item and a URL knowing that it'll be deleted in the future.

Let me know if you think otherwise and I can move this to its own page.

Reply:
I am ok with that, just add a reference link into the documentation column that points to the details below, so people do not miss this.
 "support-preview-features.md#streaming-for-http-service-invocation"

Reply:
Done